CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+* Remove support of Tarantool versions that have no builtin tuple-keydef
+  module.
+* Tarantool versions 2.{4,5,6} have reached EOL and removed from regression
+  testing. It means that methods `crud.min()` and `crud.max` won't work for old
+  1.10 versions anymore and it's a bit inconsistent with select that still
```

Comment:
> it's a bit inconsistent with select that still
  works for such versions

It was my opinion :) You shouldn't copy-paste it as is) It looks strange.

Reply:
updated